app/rule/new/NewRule.tsx

```diff
@@ -44,7 +44,22 @@ export default function NewRule() {
           <Form.Item
             label="File path/name"
             name="goRulesJSONFilename"
-            rules={[{ required: true, message: "Please input your JSON file path!" }]}
+            rules={[
```

Comment:
Great clarification for the naming requirements. The active validation is so great! Would additional helper text indicating the formatting requirements ahead of time also be helpful? Something like:
help="example: my-rule.json or my-path/my-rule.json"

Reply:
Yeah, good idea. For some reason the "help" parameter overrides any error messages, so I'm going to add it as a tooltip because that's easy